dds_web/api/project.py

```diff
@@ -480,6 +480,7 @@ def format_project_dict(self, current_user):
                 "PI": p.pi,
                 "Status": p.current_status,
                 "Last updated": p.date_updated if p.date_updated else p.date_created,
+                "Is Active": p.is_active,
```

Comment:
I think we should only return the active ones. So something like 
```
get_all = flask.request.json.get("all", False)
for p in current_user.projects:
    if not get_all and p.is_active: 
        project_info = {
            "Project ID": etc etc 
        }
```

Reply:
Also, this option overrides the builtin `all` in Python so we should think of something else again...

Reply:
@valyo one sec I have a suggestion I think that will make it more efficient

Reply:
```suggestion
        # Get info for projects
        get_all = flask.request.json.get("get_all")
        all_filters = [] if get_all else [models.Project.is_active == True]
        all_filters.append(models.Project.public_id.in_([x.public_id for x in current_user.projects]))
        user_projects = models.Project.query.filter(
                *all_filters
        ).all()
        for p in user_projects:
            project_info = {
                "Project ID": p.public_id,
                "Title": p.title,
                "PI": p.pi,
                "Status": p.current_status,
                "Last updated": p.date_updated if p.date_updated else p.date_created,
            }
```